joomla/joomla-cms
# [4.0] Menus Manager: Fix system links menu items Status icon and adding badge when component is disabled

Pull Request for Issue #24721
Replaces https://github.com/joomla/joomla-cms/pull/25185

### Summary of Changes
Correcting System Links menu items icon
Adding a badge to inform user when a component is disabled.


### Testing Instructions
Make sure you have created a System Links menu item (URL, Heading, Separator, Alias).
Create a Newsfeeds menu item and disable the Newsfeeds component in Extensions->Manage
Parameter a menu item to be published in the future.
Parameter a menu item to stop publishing in the past


### Before patch
Clicking on the status icon of a system links menu item has no effect
No way to know if the component of an existing menu item is disabled or not.


### After patch
Contrary to https://github.com/joomla/joomla-cms/pull/25185 publish_down and publish-up are respected.
If a component is disabled, the Manager will now display a badge "Component disabled"


<img width="590" alt="Screen Shot 2019-06-13 at 13 53 17" src="https://user-images.githubusercontent.com/869724/59431620-9b0f2900-8de5-11e9-893f-5852811a39f9.png">


### Note
Status icon tip as well as title tip display are not modified as this requires love and care in another PR.

@810 @SharkyKZ
